pkg/pipeline/extract/conntrack/conntrack.go

```diff
@@ -160,8 +160,10 @@ func (ct *conntrackImpl) prepareHeartbeatRecords() []config.GenericMap {
 
 func (ct *conntrackImpl) updateConnection(conn connection, flowLog config.GenericMap, flowLogHash totalHashType) {
 	d := ct.getFlowLogDirection(conn, flowLogHash)
```

Comment:
I think the call to `getFlowLogDirection()` could be moved under the new "if isn't duplicated"

Reply:
flaky tests on data race => https://github.com/netobserv/flowlogs-pipeline/actions/runs/4185399722/jobs/7252391870